### PR TITLE
Add hostname and set Locale, fixes #311

### LINKED ISSUE #311: RabbitMQ errors
Thank you for the template.  I successfully built the image in Openshift 3.11

In the RabbitMq pod logs, I see this:
```
/usr/local/bin/docker-entrypoint.sh: line 182: hostname: command not found
--
  | /usr/local/bin/docker-entrypoint.sh: line 182: hostname: command not found
```

And the health check seems to fail with this message:

```
Readiness probe failed: warning: the VM is running with native name encoding of latin1 which may cause Elixir to malfunction as it expects utf8. Please ensure your locale is set to UTF-8 (which can be verified by running "locale" in your shell) Error: this command requires the 'rabbit' app to be running on the target node. Start it with 'rabbitmqctl start_app'. Arguments given: check_port_connectivity [1mUsage[0m rabbitmq-diagnostics [--node <node>] [--longnames] [--quiet] check_port_connectivity [--timeout <timeout>]
```

Also running commands in the terminal state the locale is wrong.

Output in pod from: sh-4.2$ locale
LANG=
LC_CTYPE="POSIX"
LC_NUMERIC="POSIX"
LC_TIME="POSIX"
LC_COLLATE="POSIX"
LC_MONETARY="POSIX"
LC_MESSAGES="POSIX"
LC_PAPER="POSIX"
LC_NAME="POSIX"
LC_ADDRESS="POSIX"
LC_TELEPHONE="POSIX"
LC_MEASUREMENT="POSIX"
LC_IDENTIFICATION="POSIX"
LC_ALL=

AND:

sh-4.2$ cat /etc/locale.conf 
LANG="en_US.UTF-8"
sh-4.2$ 

Running rabbitmqctl in shell produces this warning:

warning: the VM is running with native name encoding of latin1 which may cause Elixir to malfunction as it expects utf8. Please ensure your locale is set to UTF-8 (which can be verified by running "locale" in your shell)

### PR DESCRIPTION
**Pull request by pabrahamsson**
#### What is this PR About?
`hostname` is no longer part of the base image and needs to be explicitly added.
Also, it is recommended to have the locale set to `UTF-8`

#### How do we test this?
Follow instructions in README for building and deploying. Once deployed, from the `rabbitmq` directory
```
../_test/rabbitmq.sh test rabbitmq
Ensure build is executed...
Waiting for all builds to start...
Waiting for build to complete...
Check to see if build failed
Waiting for for all pods to start...
Ensure 'hostname' package is installed
OK: RPM package 'hostname' installed
Ensure locale is set to UTF-8
OK: Locale is set to 'en_US.UTF-8'
Check RabbitMQ cluster status...
OK: no RabbitMQ cluster alarms
OK: there are 3 running nodes in the cluster
Tests Completed Successfully!
```

cc: @redhat-cop/day-in-the-life

### CODE REVIEW

File: _test/rabbitmq.sh [hunk 1]
```diff
@@ -49,6 +49,26 @@ test() {
   echo "Waiting for for all pods to start..."
   try_until 60 15 true "oc get sts/rabbitmq -n $NAMESPACE --template '{{ if .status.readyReplicas }}{{ eq .spec.replicas .status.readyReplicas }}{{ else }}false{{ end }}'" || kill -s TERM $TOP_PID
 
+  # Tests for issue #311
+  echo "Ensure 'hostname' package is installed"
+  hostname_rpm=$(oc -n $NAMESPACE rsh rabbitmq-0 rpm -q hostname)
+  if [ $? -eq 0 ]; then
```
Comment (garethahealy) [inline _test/rabbitmq.sh:55]: for this type of testing (i.e.: checking packages are installed correctly), should we start using bats?
- https://github.com/sstephenson/bats

Comment (pabrahamsson) [reply _test/rabbitmq.sh:55]: I'm definitely open to using bats, I've used it on other projects and have had a positive experience. Can you open a separate issue for this @garethahealy?

**Comment by pabrahamsson** [comment]
Thank @garethahealy, did you want bats included as part of this PR or is that a separate effort?
If the latter, can we get this one merged?

**Comment by garethahealy** [comment]
no, it's fine for another PR.

**Pull request merged**